Styles/styles.css

```diff
@@ -0,0 +1,46 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.flex-container {
+  display: flex;
+  background-color: #10367c;
+  width: 1440px;
+  height: 630px;
+  color: #ffffff;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+.item {
+  width: 33.3333%;
+  margin-top: 120px;
+
+}
+li {
+  list-style-type: none;
+  line-height: 24px;
+  margin-bottom: 24px;
+}
+.about {
+  margin-left: 104px;
+}
+
+ul {
+  line-height: 2;
+}
+ul li:first-child {
+  font-size: 20px;
+}
+.trip {
+  position: fixed;
```

Comment:
There is no reason to prevent the document's [normal flow](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Normal_Flow). Read about [positioning](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Positioning) and learn the cases when to use it.